frontend/src/components/DeliveryDetails.tsx

```diff
@@ -21,6 +22,11 @@ export default function DeliveryDetails(props: DetailsProps) {
         //eslint-disable-next-line
```

Comment:
könnte man hier eslint nicht wieder aktivieren da ja in der liste die id steht ? Wir haben es auch so aber weil in der Liste kein wert steht ?